py-polars/polars/internals/frame.py

```diff
@@ -4251,13 +4251,15 @@ def n_unique(self) -> DataFrame:
             return wrap_df(self._df.downsample(self.by, self.rule, self.n, "n_unique"))
         return wrap_df(self._df.groupby(self.by, self.selection, "n_unique"))
 
-    def quantile(self, quantile: float) -> DataFrame:
+    def quantile(self, quantile: float, interpolation: str = "nearest") -> DataFrame:
```

Comment:
Let's add a parameters sections which explains the possible interpolation strings